src/quick-lint-js/have.h

```diff
@@ -312,15 +312,6 @@
 #define QLJS_HAVE_FTS_H 0
 #endif
 
-#if !defined(QLJS_HAVE_SETJMP) && defined(__has_include)
-#if !defined(__EMSCRIPTEN__) && __has_include(<csetjmp>)
-#define QLJS_HAVE_SETJMP 1
-#endif
-#endif
-#if !defined(QLJS_HAVE_SETJMP)
-#define QLJS_HAVE_SETJMP 0
-#endif
```

Comment:
Must fix: We should use have.h and not exclusively rely on CMake configure checks. (CMake configure checks are fine *in addition to* have.h.) Explanation: https://github.com/quick-lint/quick-lint-js/blob/28d647a2090d991cc686c17c52aa26519d5e4883/docs/architecture/ADR001-Feature-testing-with-have-h.md